internal/publish/publish.go

```diff
@@ -267,49 +265,6 @@ func (p *defaultPublisher) createDeploymentRecord(
 	return p.writeDeploymentRecord()
 }
 
-// Handle preflight agent error details
-func (p *defaultPublisher) preflightAgentError(agenterr *types.AgentError, contentID types.ContentID) *types.AgentError {
-	agenterr.Code = events.DeploymentFailedCode
-
-	if aerr, isNotFound := http_client.IsHTTPAgentErrorStatusOf(agenterr, http.StatusNotFound); isNotFound {
-		p.log.Debug("Content cannot be found. Deployment cannot proceed", "content_id", contentID)
-		aerr.Message = fmt.Sprintf("Cannot deploy content: ID %s - Content cannot be found", contentID)
-	} else if aerr, isForbidden := http_client.IsHTTPAgentErrorStatusOf(agenterr, http.StatusForbidden); isForbidden {
-		p.log.Debug("Insufficient permissions. Content deployment cannot proceed", "content_id", contentID)
-		aerr.Message = fmt.Sprintf("Cannot deploy content: ID %s - You may need to request collaborator permissions or verify the credentials in use", contentID)
-	} else {
-		p.log.Debug("Unknown error while deploying", "content_id", contentID)
-		aerr.Message = fmt.Sprintf("Cannot deploy content: ID %s - Unknown error: %s", contentID, agenterr.Error())
-	}
-
-	return agenterr
-}
-
```

Comment:
Below you'll see the original code which was moved into client_connect: preflightAgentError method.